src/Umbraco.Core/Notifications/NotificationExtensions.cs

```diff
@@ -1,3 +1,5 @@
+using Umbraco.Cms.Core.Models;
+
 namespace Umbraco.Cms.Core.Notifications;
```

Comment:
This seems unnecessary 😁 
```suggestion
namespace Umbraco.Cms.Core.Notifications;
```

Reply:
🙈 leftover i forgot to clean up